activesupport/lib/active_support/core_ext/erb/util.rb

```diff
@@ -1,6 +1,31 @@
 # frozen_string_literal: true
 
 class ERB
+  module ActiveSupportUtil
```

Comment:
I think that module should be `ActiveSupport::CoreExt::ERBUtil`

Reply:
Fixed it and pushed https://github.com/rails/rails/pull/46573/commits/0fc2c6f2100cc91560dc3262b64172ac62f866e3.